fandogh_cli/cluster_commands.py

```diff
@@ -3,7 +3,7 @@
 
 from .config import *
 
-fandogh = [{'name': 'fandogh', 'url': 'https://api.fandogh.cloud', 'active': True}]
+fandogh = [{'name': 'fandogh', 'url': 'fandogh.cloud', 'active': True}]
```

Comment:
what happens to old version which have fandogh https://api.fandogh.cloud in credentials file?